backend/app/api/routes/documents.py

```
@@ -224,29 +258,25 @@ async def process_multiple_documents(
                 detail=f"File '{file.filename}' exceeds the {MAX_FILE_SIZE_MB}MB size limit.",
             )
 
-        title_without_extension = os.path.splitext(file.filename)[0]
+        filename_str = file.filename if file.filename is not None else ""
```

nit: will there be an instance filename will be none, if so, should we throw an error since that isn't an ideal case